msteams-platform/TOC.yml

```diff
@@ -48,6 +48,8 @@
       href: concepts/device-capabilities/device-capabilities-overview.md
```

Comment:
The overview should mention device permission are different in the browser and reference the browser doc page.